ports/servo/platform/macos/count_threads.c

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <mach/mach.h>
+
+int count_threads() {
```

Comment:
lets name this something more verbose to avoid symbol clashes in the future

Reply:
Renamed to `macos_count_running_threads`.